ProcessLib/LIE/SmallDeformation/SmallDeformationProcess.h

```diff
@@ -45,6 +44,9 @@ class SmallDeformationProcess final : public Process
     bool isLinear() const override;
     //! @}
 
+    void computeSecondaryVariableConcrete(double const t,
```

Comment:
Sorry for the late comment. AFAIK `computeSecondaryVariableConcrete()` is called after every single iteration.
In contrast `postTimestepConcreteProcess()` is called only after each timestep, so probably much less often. You seem to use the secondary variables only for output. Wouldn't `postTimestepConcreteProcess()` therefore suffice?
The only corner case that I am aware of is the output of iteration results. Or am I missing something?
For later developments, of course a `preOutput()` method would be a good option.

Reply:
I thought it would be the other way around, but it looks like the compute secondary vars is called just after postTimestep [UncoupledTimeLoop.cpp](https://github.com/ufz/ogs/blob/master/ProcessLib/UncoupledProcessesTimeLoop.cpp#L684).

I agree with the need for preOutput(). --> another PR.

I'd suggest to keep this implementation for now.

Reply:
Yep, implementation can be kept. Then I was wrong. Maybe then in the future the two methods can be combined.